src/bindings.cpp

```diff
@@ -770,10 +779,10 @@ extern "C"
     /// [handle]
     FFI_PLUGIN_EXPORT void stop(unsigned int handle)
     {
-        if (player.get() == nullptr || !player.get()->isInited() ||
-            !player.get()->isValidHandle(handle))
```

Comment:
Wait, why are we no longer checking whether the handle is valid? Does `Player::stop()` not care?

Oh wait, we checked whether the handle was _not_ valid? And then we stopped it? Sorry, I'm getting confused.

Reply:
Thanks for the explanation!